src/config-schemas/openmrs-esm-patient-registration-schema.ts

```diff
@@ -0,0 +1,34 @@
+import { validators } from '@openmrs/esm-module-config';
+export const esmPatientRegistrationSchema = {
+  personAttributes: {
+    arrayElements: {
+      subTitle: { validators: [validators.isString] },
+      label: { validators: [validators.isString] },
+      uuid: { validators: [validators.isString] },
```

Comment:
Use the `isUuid` validator. Check out the available validators: https://github.com/openmrs/openmrs-esm-module-config/blob/master/README.md#const-validators

Reply:
Fixed